src/RMP/Translate/Translate.php

```diff
@@ -90,6 +90,12 @@ public function translate($key, $substitutions = array(), $pluralisation = null,
         }
         $result = $this->_translate($key, $substitutions, $pluralisation, $domain, $this->locale);
 
+        // test if the locale is the 2 letter variant which is oftent taken from ckps_language
+        $gbLocales = ['en','cy','gd','ga'];
```

Comment:
I may be having a foggy memory here, but is this used in Amen?
If so, then it'll need to support PHP 5.3, so `array()`